### PR TITLE
MINOR: Update files with 3.1.0

### PR DESCRIPTION
**Pull request by dajac**
Add 3.0.0 and 3.1.0 in various places. I will follow up with other PRs to update the compatibility tests.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: tests/docker/Dockerfile [hunk 1]
```diff
@@ -80,6 +82,8 @@ RUN curl -s "$KAFKA_MIRROR/kafka-streams-2.5.1-test.jar" -o /opt/kafka-2.5.1/lib
 RUN curl -s "$KAFKA_MIRROR/kafka-streams-2.6.2-test.jar" -o /opt/kafka-2.6.2/libs/kafka-streams-2.6.2-test.jar
 RUN curl -s "$KAFKA_MIRROR/kafka-streams-2.7.1-test.jar" -o /opt/kafka-2.7.1/libs/kafka-streams-2.7.1-test.jar
 RUN curl -s "$KAFKA_MIRROR/kafka-streams-2.8.1-test.jar" -o /opt/kafka-2.8.1/libs/kafka-streams-2.8.1-test.jar
+RUN curl -s "$KAFKA_MIRROR/kafka-streams-3.0.0-test.jar" -o /opt/kafka-2.8.1/libs/kafka-streams-3.0.0-test.jar
```
Comment (bbejeck) [inline tests/docker/Dockerfile]: I think this should be `/opt/kafka-3.0.0/...`

File: tests/docker/Dockerfile [hunk 2]
```diff
@@ -80,6 +82,8 @@ RUN curl -s "$KAFKA_MIRROR/kafka-streams-2.5.1-test.jar" -o /opt/kafka-2.5.1/lib
...
+RUN curl -s "$KAFKA_MIRROR/kafka-streams-3.1.0-test.jar" -o /opt/kafka-2.8.1/libs/kafka-streams-3.1.0-test.jar
```
Comment (bbejeck) [inline tests/docker/Dockerfile]: same here `/opt/kafka-3.1.0/`

**Review by bbejeck — COMMENTED** [review]
Thanks @dajac one minor comment otherwise this LGTM

**Comment by dajac** [comment]
@bbejeck Thanks. Updated.

**Review by bbejeck — APPROVED** [review]

**Pull request merged**